src/mtcp/mtcp_util.ic

```diff
@@ -243,6 +243,23 @@ void mtcp_mkdir(const char *dir)
   mtcp_sys_mkdir(tmp, S_IRWXU);
 }
 
+int mtcp_readvfile(int fd, struct iovec *iov, int iovcnt)
+{
+  int i = 0;
+  int count = 0;
+  int readC = 0;
+
+  while (i < iovcnt) {
+    readC = mtcp_readfile(fd, iov[i].iov_base, iov[i].iov_len);
+    if (readC != iov[i].iov_len) {
+      return count + readC;
+    }
+    count += readC;
+    i++;
+  }
+  return 0;
```

Comment:
Shall we return `count` here? Also, I think it's better to make the return type `size_t`.